### DISCUSSION TITLE
Support for LNURL-withdraw for LNBank wallet

### DISCUSSION THREAD

**Discussion by ishristov** [btcpayserver/btcpayserver#3883] [Ideas / Feature Requests]
This is inspired by the CoinCorner bolt cards that can be configured to withdraw BTC via lnurl-withdraw from a specific CoinCorner KYC account when it's being nfc-read by a supported merchant.

It would be nice if we could configure such a card (or any other nfc tag) to withdraw BTC from a LNBank wallet. From what I understand, this should somehow happen via the LNURL-withdraw but I'm not 100% sure what needs to be done in order for that to work. If 

If that's possible, a user can have a LNBank wallet that receives LN payments from their store and then will be able to use the configured nfc-based card on the go to pay at other merchants directly from that LNBank wallet.

I'd be happy to hear some feedback and being corrected if I'm misunderstanding things.

**Comment by dennisreimann** [comment]
Let's discuss this here: https://github.com/dennisreimann/btcpayserver/issues/12
